lib/active_model_serializers/adapter/json_api/relationship.rb

```diff
@@ -34,6 +34,11 @@ def as_json
 
         private
 
+        def ensure_valid!(json)
```

Comment:
Merging this PR would still validate against the existing schema 😄 

But I am fine with waiting, as long as we verify there are no other concerns.